src/transformation/visitors/class/index.ts

```diff
@@ -65,28 +68,19 @@ interface ClassSuperInfo {
     extendedTypeNode?: ts.ExpressionWithTypeArguments;
 }
 
-export function transformClassDeclaration(
+function transformClassLikeDeclaration(
     classDeclaration: ts.ClassLikeDeclaration,
     context: TransformationContext,
```

Comment:
For consistency:
```suggestion
    context: TransformationContext,
    classDeclaration: ts.ClassLikeDeclaration,
```

Reply:
Actually it is consistent right now with transformClassDeclaration and transformClassAsExpression